### PR TITLE
#184581192 Add items to cart

### PR DESCRIPTION
**Pull request by khevin2**
#### What does this PR do?
This PR allows a user/buyer to add products they wish to buy in shopping cart

#### Description of Task to be completed?
- Create cart creation routes
- Update if the cart already exist
- Create docs and tests for the functionalities above.

#### How should this be manually tested?
- Clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-bn`
- Switch your working directory `cd e-comm-team-amigos-bn`
- Create an `.env` file and update it with `.env-example` file
- Install all the dependencies by running `npm install`
- Then run the project by typing `npm start`

#### Endpoints to be tested in Postman:
- `GET` `/cart/?productId={productId}&quantity={quantity}` to update a product.
- Here, `productId` is the `ID` of th product trying to add to cart.

#### Any background context you want to provide?
- Have Node Js installed in your local machine
- Have Postgres
- API tester like Postman

#### What are the relevant pivotal tracker?

Story ID: [#184581192](https://www.pivotaltracker.com/n/projects/2632829/stories/184581192)

#### Screenshots 
Screenshot from 2023-03-23 13-54-44

Screenshot from 2023-03-23 14-06-07

Screenshot from 2023-03-23 15-19-14

### CODE REVIEW

File: src/controllers/cart.controller.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import models from "./../database/models";
+
+export async function createCart(req, res) {
+    try {
+        const reqCart = req.body.cart;
+        const owner = req.params.ownerId;
+        console.log("\nowner\n", owner);
```
Comment (Emile-SHUMBUSHO) [inline src/controllers/cart.controller.js]: console.log should not pushed to the version control

**Comment by Emile-SHUMBUSHO** [comment]
From my local machine I can't find updated code from develop, remember to rebase your branch from develop

File: src/controllers/cart.controller.js (hunk 1 above)
Comment (khevin2) [reply src/controllers/cart.controller.js]: I'm still working on it

**Comment by khevin2** [comment]
> From my local machine I can't find updated code from develop, remember to rebase your branch from develop

I have rebased this morning. Check well.

**Review by GasanaJr — CHANGES_REQUESTED** [review]
Modify the PR body to contain screenshots of the passing tests and the swagger documentation. Resolve the conflicting files with develop

**Review by AbdulKhaliq59 — APPROVED** [review]

**Review by AbdulKhaliq59 — CHANGES_REQUESTED** [review]
Resolve conversation and conflict

**Review by AbdulKhaliq59 — APPROVED** [review]

**Review by GasanaJr — APPROVED** [review]

**Review by GasanaJr — APPROVED** [review]
LGTM

**Review by wilbrord2 — APPROVED** [review]

**Review by ericndungutse — APPROVED** [review]
LGTM

**Review by Emile-SHUMBUSHO — APPROVED** [review]

**Review by Mystica52 — APPROVED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**